### DISCUSSION TITLE
Discord Server?

### DISCUSSION THREAD

**Discussion by GageSorrell** [Sixze/ALS-Refactored#85] [Q&A]
Hello,

I have an intermediate level experience in programming in Unreal, but I know very little about animations.  Your refactor seems to implement a lot of good patterns that the Community version does not, and I am struggling to understand ALS and your changes as well.

Would you consider creating a Discord server where people can ask questions regarding your plugin?

Thank you,
Gage

**Comment by Sixze** [comment]
There is no separate Discord server for my ALS version, but you can ask your questions here: https://discord.gg/wYYMHFu
